www/docs/en/9.x/guide/appdev/hooks/index.md

```diff
@@ -383,6 +377,15 @@ If a script returns a non-zero exit code, then the parent cordova command will b
 
 > __Note__: we highly recommend writing your hooks using Node.js so that they are cross-platform, see [Javascript](#link-javascript) section above.
 
+#### Windows Quirks
+
+If you are working on Windows, and your hook scripts aren't `*.bat` files, Cordova CLI will expect a shebang line as the first line of the script. This way it knows the interpreter it needs to use to launch the script. A shebang line for a Python script could look like this:
+
+```
+#!/usr/bin/env python
+```
+
+
```

Comment:
Don't need two newlines, but also not necessary to make any effort to change this.